app/src/main/res/layout/activity_bookingpage.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For this booking page, the button is cut out and the page doesn't seem to scroll, so there is no way to access it.

![image](https://user-images.githubusercontent.com/44113229/170635641-40caa345-8d6c-42e2-ba3f-6dd64a029df6.png)